spec/sidekiq_flow/client_spec.rb

```diff
@@ -102,6 +95,11 @@
         subject
         expect(Sidekiq::Worker.jobs.last['args'][1]).to eq(task_klass)
       end
+
```

Comment:
For the client_spec, please add specs to all public methods.

e.g for store workflow:
```
describe '.store_workflow' do
  it 'should work correctly' do
    subject
    expect(expect(SidekiqFlow::Client.find_workflow('some-id')).to eq(something)
  end
end
```